ehealthid-rp/src/main/resources/i18n.properties

```diff
@@ -0,0 +1,5 @@
+title=Login with GesundheitsID
+error.message=Ohhh noes!
```

Comment:
I would use something more serious...

Reply:
I keep the one from Thomas, but sure we can go with something more meaningful